contrib/kea/files/kea-dhcp4

```diff
@@ -0,0 +1,6 @@
+# kea-dhcp4 service
+
+type       = process
+command    = /usr/sbin/kea-dhcp4 -c /etc/kea/kea-dhcp4.conf
+depends-on = network.target
+after      = time-sync.target
```

Comment:
```suggestion
type = process
command = /usr/sbin/kea-dhcp4 -c /etc/kea/kea-dhcp4.conf
depends-on = network.target
after = time-sync.target
```

don't need alignment anymore for all of these

Reply:
good to know.  Thanks!